client/src/html.tsx

```diff
@@ -17,8 +18,16 @@ export default function HTML({
   postBodyComponents,
   preBodyComponents
 }: HTMLProps): JSX.Element {
+  const rtlDirectionAttribute = { dtr: 'rtl' };
+  const rtlLanguage: boolean = ['arabic', 'udru'].includes(clientLocale);
```

Comment:
Please move this to the configs

Reply:
Couldn't find them, or do you mean config.json?

Reply:
Use `/config/i18n/all-langs.ts`